torch/nn/modules/sparse.py

```diff
@@ -75,7 +75,7 @@ class Embedding(Module):
 
     def __init__(self, num_embeddings, embedding_dim, padding_idx=None,
                  max_norm=None, norm_type=2, scale_grad_by_freq=False,
-                 sparse=False):
+                 sparse=False, weight=None):
```

Comment:
Can you change the `weight` argument to `_weight` (such that it is for internal usage only)?